Makefile

```diff
@@ -129,6 +140,10 @@ install:
 	glide install
 
 
+.PHONY: prototest
+prototest:
```

Comment:
Ya I can remove this, this was just for testing myself, it's helpful for now but ya